NebulaNetwork/PacketProcessors/Logistics/ILSIdleShipBackToWorkProcessor.cs

```diff
@@ -16,18 +17,18 @@ public class ILSIdleShipBackToWorkProcessor : PacketProcessor<ILSIdleShipBackToW
 {
     protected override void ProcessPacket(ILSIdleShipBackToWork packet, NebulaConnection conn)
     {
-        if (IsHost)
+        if (!IsClient)
         {
             return;
         }
 
-        if (!IsClient)
+        try
         {
-            return;
+            ILSShipManager.IdleShipGetToWork(packet);
         }
-        using (Multiplayer.Session.Factories.IsIncomingRequest.On())
```

Comment:
Is removing this using also intentional?

Reply:
yes, it's intentional.
There isn't methods on the call path that require the locking of `Factories.IsIncomingRequest`